### ISSUE TITLE
Can't find linked libraries on Ubuntu 20.04

### ISSUE THREAD

**Issue by EpicWink** [CloudCompare/CloudCompare#1294]
CloudCompare can't find some of its dynamic libraries in my fresh install of Ubuntu 20.04

It works if I set `LD_LIBRARY_PATH=/usr/local/bin`

### Reproduction
1. Clone, build and install CloudCompare
2. Run `CloudCompare`

### Your environment
* CC: a1c589c
  * CCCoreLib: 0fd2704
* OS: Ubuntu 20.04 (installed hours earlier)
* Qt: 5.12.8 (Ubuntu apt)

`echo $PATH`:
```
~/bin:~/.local/bin:~/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
```

`echo $LD_LIBRARY_PATH`:
```
```

`ls /usr/local/bin`:
```
CloudCompare  libCCAppCommon.so  libCCCoreLib.so  libCC_FBO_LIB.so  libCCPluginAPI.so  libQCC_DB_LIB.so  libQCC_GL_LIB.so  libQCC_IO_LIB.so
```

`ls /usr/local/lib/cloudcompare/`:
```
plugins
```

<details>
<summary><code>cmake -LA</code> (click to expand)</summary>

```
-- OpenMP found
-- Install shared library: CCAppCommon
-- Install shared library: CC_FBO_LIB
-- Install shared library: CCPluginAPI
-- Install shared library: CCCoreLib
-- Install shared library: QCC_DB_LIB
-- Install shared library: QCC_GL_LIB
-- Install shared library: QCC_IO_LIB
-- Added io plugin: QADDITIONAL_IO_PLUGIN
-- Added io plugin: QCORE_IO_PLUGIN
-- Added io plugin: QPDAL_IO_PLUGIN
-- Added standard plugin: QANIMATION_PLUGIN
-- Install plugins
--  Types: gl;io;standard
--  Destination: lib/cloudcompare/plugins/
--  Shader Destination: share/cloudcompare/shaders/
--  Install QADDITIONAL_IO_PLUGIN (io)
--  Install QCORE_IO_PLUGIN (io)
--  Install QPDAL_IO_PLUGIN (io)
--  Install QANIMATION_PLUGIN (standard)
-- Configuring done
-- Generating done
-- Build files have been written to: ~/src/cloudcompare/build
-- Cache values
BUILD_TESTING:BOOL=OFF
CCACHE_PROGRAM:FILEPATH=/usr/bin/ccache
CCCORELIB_SCALAR_DOUBLE:BOOL=OFF
CCCORELIB_SHARED:BOOL=ON
CCCORELIB_USE_CGAL:BOOL=OFF
CCCORELIB_USE_QT_CONCURRENT:BOOL=ON
CCCORELIB_USE_TBB:BOOL=ON
CMAKE_ADDR2LINE:FILEPATH=/usr/bin/addr2line
CMAKE_AR:FILEPATH=/usr/bin/ar
CMAKE_BUILD_TYPE:STRING=Release
CMAKE_COLOR_MAKEFILE:BOOL=ON
CMAKE_CXX_COMPILER:FILEPATH=/usr/bin/c++
CMAKE_CXX_COMPILER_AR:FILEPATH=/usr/bin/gcc-ar-9
CMAKE_CXX_COMPILER_RANLIB:FILEPATH=/usr/bin/gcc-ranlib-9
CMAKE_CXX_FLAGS:STRING=
CMAKE_CXX_FLAGS_DEBUG:STRING=-g
CMAKE_CXX_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG
CMAKE_CXX_FLAGS_RELEASE:STRING=-O3 -DNDEBUG
CMAKE_CXX_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG
CMAKE_C_COMPILER:FILEPATH=/usr/bin/cc
CMAKE_C_COMPILER_AR:FILEPATH=/usr/bin/gcc-ar-9
CMAKE_C_COMPILER_RANLIB:FILEPATH=/usr/bin/gcc-ranlib-9
CMAKE_C_FLAGS:STRING=
CMAKE_C_FLAGS_DEBUG:STRING=-g
CMAKE_C_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG
CMAKE_C_FLAGS_RELEASE:STRING=-O3 -DNDEBUG
CMAKE_C_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG
CMAKE_DLLTOOL:FILEPATH=CMAKE_DLLTOOL-NOTFOUND
CMAKE_EXE_LINKER_FLAGS:STRING=
CMAKE_EXE_LINKER_FLAGS_DEBUG:STRING=
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_EXE_LINKER_FLAGS_RELEASE:STRING=
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_EXPORT_COMPILE_COMMANDS:BOOL=OFF
CMAKE_INSTALL_BINDIR:PATH=bin
CMAKE_INSTALL_DATADIR:PATH=
CMAKE_INSTALL_DATAROOTDIR:PATH=share
CMAKE_INSTALL_DOCDIR:PATH=
CMAKE_INSTALL_INCLUDEDIR:PATH=include
CMAKE_INSTALL_INFODIR:PATH=
CMAKE_INSTALL_LIBDIR:PATH=lib
CMAKE_INSTALL_LIBEXECDIR:PATH=libexec
CMAKE_INSTALL_LOCALEDIR:PATH=
CMAKE_INSTALL_LOCALSTATEDIR:PATH=var
CMAKE_INSTALL_MANDIR:PATH=
CMAKE_INSTALL_OLDINCLUDEDIR:PATH=/usr/include
CMAKE_INSTALL_PREFIX:PATH=/usr/local
CMAKE_INSTALL_RUNSTATEDIR:PATH=
CMAKE_INSTALL_SBINDIR:PATH=sbin
CMAKE_INSTALL_SHAREDSTATEDIR:PATH=com
CMAKE_INSTALL_SYSCONFDIR:PATH=etc
CMAKE_LINKER:FILEPATH=/usr/bin/ld
CMAKE_MAKE_PROGRAM:FILEPATH=/usr/bin/make
CMAKE_MODULE_LINKER_FLAGS:STRING=
CMAKE_MODULE_LINKER_FLAGS_DEBUG:STRING=
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_MODULE_LINKER_FLAGS_RELEASE:STRING=
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_NM:FILEPATH=/usr/bin/nm
CMAKE_OBJCOPY:FILEPATH=/usr/bin/objcopy
CMAKE_OBJDUMP:FILEPATH=/usr/bin/objdump
CMAKE_RANLIB:FILEPATH=/usr/bin/ranlib
CMAKE_READELF:FILEPATH=/usr/bin/readelf
CMAKE_SHARED_LINKER_FLAGS:STRING=
CMAKE_SHARED_LINKER_FLAGS_DEBUG:STRING=
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_SHARED_LINKER_FLAGS_RELEASE:STRING=
CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_SKIP_INSTALL_RPATH:BOOL=NO
CMAKE_SKIP_RPATH:BOOL=NO
CMAKE_STATIC_LINKER_FLAGS:STRING=
CMAKE_STATIC_LINKER_FLAGS_DEBUG:STRING=
CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_STATIC_LINKER_FLAGS_RELEASE:STRING=
CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_STRIP:FILEPATH=/usr/bin/strip
CMAKE_VERBOSE_MAKEFILE:BOOL=FALSE
DXF_LIB_DIR:PATH=extern/dxflib
FFMPEG_INCLUDE_DIR:PATH=/usr/include/x86_64-linux-gnu
FFMPEG_LIBRARY_DIR:PATH=/usr/lib/x86_64-linux-gnu
OPTION_BUILD_CCVIEWER:BOOL=OFF
OPTION_GL_QUAD_BUFFER_SUPPORT:BOOL=OFF
OPTION_SUPPORT_GAMEPADS:BOOL=OFF
OPTION_USE_DXF_LIB:BOOL=ON
OPTION_USE_GDAL:BOOL=OFF
OPTION_USE_OCULUS_SDK:BOOL=OFF
OPTION_USE_SHAPE_LIB:BOOL=ON
OpenMP_CXX_FLAGS:STRING=-fopenmp
OpenMP_CXX_LIB_NAMES:STRING=gomp;pthread
OpenMP_C_FLAGS:STRING=-fopenmp
OpenMP_C_LIB_NAMES:STRING=gomp;pthread
OpenMP_gomp_LIBRARY:FILEPATH=/usr/lib/gcc/x86_64-linux-gnu/9/libgomp.so
OpenMP_pthread_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libpthread.so
PDAL_DIR:PATH=/usr/lib/cmake/PDAL
PLUGIN_EXAMPLE_GL:BOOL=OFF
PLUGIN_EXAMPLE_IO:BOOL=OFF
PLUGIN_EXAMPLE_STANDARD:BOOL=OFF
PLUGIN_GL_QEDL:BOOL=OFF
PLUGIN_GL_QSSAO:BOOL=OFF
PLUGIN_IO_QADDITIONAL:BOOL=ON
PLUGIN_IO_QCORE:BOOL=ON
PLUGIN_IO_QCSV_MATRIX:BOOL=OFF
PLUGIN_IO_QE57:BOOL=OFF
PLUGIN_IO_QFBX:BOOL=OFF
PLUGIN_IO_QPDAL:BOOL=ON
PLUGIN_IO_QPHOTOSCAN:BOOL=OFF
PLUGIN_IO_QRDB:BOOL=OFF
PLUGIN_STANDARD_QANIMATION:BOOL=ON
PLUGIN_STANDARD_QBROOM:BOOL=OFF
PLUGIN_STANDARD_QCANUPO:BOOL=OFF
PLUGIN_STANDARD_QCOMPASS:BOOL=OFF
PLUGIN_STANDARD_QCORK:BOOL=OFF
PLUGIN_STANDARD_QCSF:BOOL=OFF
PLUGIN_STANDARD_QFACETS:BOOL=OFF
PLUGIN_STANDARD_QHOUGH_NORMALS:BOOL=OFF
PLUGIN_STANDARD_QHPR:BOOL=OFF
PLUGIN_STANDARD_QM3C2:BOOL=OFF
PLUGIN_STANDARD_QPCL:BOOL=OFF
PLUGIN_STANDARD_QPCV:BOOL=OFF
PLUGIN_STANDARD_QPOISSON_RECON:BOOL=OFF
PLUGIN_STANDARD_QRANSAC_SD:BOOL=OFF
PLUGIN_STANDARD_QSRA:BOOL=OFF
QANIMATION_WITH_FFMPEG_SUPPORT:BOOL=ON
QT5_ROOT_PATH:PATH=/usr/lib/x86_64-linux-gnu/qt5
Qt5Concurrent_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5Concurrent
Qt5Core_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5Core
Qt5Gui_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5Gui
Qt5LinguistTools_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5LinguistTools
Qt5OpenGLExtensions_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5OpenGLExtensions
Qt5OpenGL_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5OpenGL
Qt5PrintSupport_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5PrintSupport
Qt5Svg_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5Svg
Qt5Widgets_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5Widgets
Qt5_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/Qt5
SHAPELIB_LIB_DIR:PATH=extern/shapelib
TBB_DIR:PATH=/usr/lib/x86_64-linux-gnu/cmake/TBB
```
</details>

**Comment by rjanvier** [comment]
Thanks. Please use v2.11.0 branch for now. CC is under refactoring in master branch and I hadn’t time to review all commits.

**Comment by andreadotti** [comment]
Libraries are installed in `bin` subdirectory instead of `lib`:
```
ls cc/bin/
CloudCompare  ccViewer  libCCAppCommon.so  libCCCoreLib.so  libCCPluginAPI.so  libCC_FBO_LIB.so  libQCC_DB_LIB.so  libQCC_GL_LIB.so  libQCC_IO_LIB.so
```
Fresh install from master

**Comment by EpicWink** [comment]
This is fixed by #1346
